pvlib/spectrum/mismatch.py

```diff
@@ -239,8 +239,9 @@ def integrate(e):
     return smm
 
 
-def spectral_factor_firstsolar(pw, airmass_absolute, module_type=None,
-                               coefficients=None, min_pw=0.1, max_pw=8):
+def spectral_factor_firstsolar(precipitable_water, airmass_absolute,
+                               module_type=None, coefficients=None,
+                               min_pw=0.1, max_pw=8):
```

Comment:
Perhaps a bit strange that we still use `pw` in `min_pw`  and `max_pw` - so in essence users are still required to know the acronym